ospf6d/ospf6_top.c

```diff
@@ -2030,12 +2035,18 @@ ospf6_show_summary_address(struct vty *vty, struct ospf6 *ospf6,
 			json_object_string_add(json_aggr,
 					"Summary address",
 					buf);
+			json_object_string_add(json_aggr, "summaryAddress",
+					       buf);
 
 			json_object_string_add(
 				json_aggr, "Metric-type",
 				(aggr->mtype == DEFAULT_METRIC_TYPE)
 					? "E2"
 					: "E1");
+			json_object_string_add(
+				json_aggr, "metricType",
+				(aggr->mtype == DEFAULT_METRIC_TYPE) ? "E2"
+								     : "E1");
 
 			json_object_int_add(json_aggr, "Metric",
```

Comment:
How about also cleaning up keys starting with capital letter like `Metric` and `Tag` in this file?

Reply:
Fixed `PascalCase` too.